### DISCUSSION TITLE
[Weekly Meeting] [2024-07-26] Meeting Minutes

### DISCUSSION THREAD

**Discussion by porcelli** [apache/incubator-kie-issues#1407] [Weekly Meetings]
- Repositories part of the release
  - Documents?
  - Website?
  - @domhanak asking to have branches in the docs repos
  - @porcelli mentioned that nothing has been covered in docs: no automation, no packaging, even configuration and hosting are not addressed at this point
  - Doc Repos
     - https://github.com/apache/incubator-kie-kogito-docs/issues
     - https://docs.kogito.kie.org/latest/html_single/
     - https://github.com/apache/incubator-kie-docs
     - https://sonataflow.org/serverlessworkflow/main/index.html
  - @jstastny-cz just update versions, this is going to be problematic. several compoenents are deprecated or renamed.
  - @tiagobento past time to change release
  - Website
     - should be updated once binaries are available
- Reverting circular dependency
  - @porcelli quick recap: [updated after call]
  - @wmedvede raising concerned that this need only being coordinated + the fact that 10 is not yet released. completely disconnected with 10 and it is not related to 10 at all.
  - @porcelli mentioned although it's ok the current state of the codebase, we as community has to follow what has been voted
  - @domhanak the proposal wasn't a good thing to break the main. couple more weeks to `resolve the issue` also asked.
  - @tiagobento the idea of having an epic is not in compliance with the voted proposal.
  - @tiagobento @domhanak disagreeing in procedure, but @tiagobento is highlighting that this has been voted and we need to follow.
  - @porcelli and @pibizza agrees that it's fair to wait for the 10 release, as the original thread gives room to interpret that 10 RELEASE should be waited.
  - @porcelli also highlighted that although the concept to follow the spirit of the text.. to avoid bigger issues, it's better us as community to stick with what is written without much margin to different interpretation.
  - @tiagobento TL;DR: we should face the consequences of the decisions, can't revisit decisions. let's learn from those decisions as community.
- 10 release!
  - Branch done! Now working on Automation D (automation that builds the artifacts)
  - @rodrigonull working on D and should be done very soon (several PRs open). 
  - @porcelli SVN credentials are available for all committers, but not bot access.
  - @pibizza raising concern that might have 3 blockers.
  - @tiagobento highlight that release source of truth is Apache SVN. in case of publishing in different places, it needs to comply with Apache policies.
  - Blockers:
     - https://github.com/apache/incubator-kie-issues/issues/1266 -  @wmedvede clarified that we don't need operator in operator hub, users can install with additional procedures. not a blocker.
     - https://github.com/apache/incubator-kie-issues/issues/1254 -> done!
     - https://github.com/apache/incubator-kie-issues/issues/1267 -> probably done!
  - Release procedure: once we have Automation D, it starts the release process. That is followed by Voting and verification (checksum, source file headers, build from source, etc).

----
https://us06web.zoom.us/rec/share/7NIU2LuKy1JqhbpNTdsa1LCuZlk3F27UEY3hvbtGK2FVrhZLavHo_rXYRwVdw6uM.Yzkywqjln-0BNEpS
Passcode: x%jupcM7

**Comment by tkobayas** [comment]
FYI)
Doc Repos (Drools)
- https://github.com/apache/incubator-kie-drools/tree/main/drools-docs -> https://docs.drools.org/latest/drools-docs/drools/introduction/index.html
